test/get-module-specifier-test.js

```diff
@@ -59,6 +59,17 @@ describe('get-module-specifier', function() {
     );
   });
 
+  it('identifies named module with a dot delimeter', function() {
+    let moduleExtension = '.hbs';
+
+    assert.equal(getModuleSpecifier(modulePrefix, moduleConfig, 'ui/components/text-editor.glimmer', moduleExtension),
+      'template:/my-app/components/text-editor.glimmer'
```

Comment:
@rwjblue So the specifier should not include the manager name but the module name should? Troll but legit question. We would end up with a module map that looks something like:

```js
import { default as __ui__components__hello_glimmer__template_fragment__ } from './ui/components/hello-glimmer/template.fragment';

export default { 'template:/hello-glimmer/components/hello-glimmer': __ui__components__hello_glimmer__template_fragment__ };
```